components/eamxx/cime_config/testdefs/testmods_dirs/scream/output_coarsening/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange SCREAM_ATMCHANGE_BUFFER='Scorpio::output_yaml_files=${SRCROOT}/components/eamxx/data/scream_coarsening_output.yaml'
```

Comment:
You'll want xmlchange to append this value (`--append`) so as to not flush whatever might have already been in the buffer.